examples/callbacks/callbacks.go

```diff
@@ -35,6 +38,8 @@ func main() {
 	// Deleting the item will execute the AboutToDeleteItem callback.
 	cache.Delete("someKey")
 
+	//remove AddedItemCallbacks
```

Comment:
That comment feels a bit redundant :smile:

Reply:
not sure what i've thought there, I removed the comment.